client/src/components/Profile/EditProfile.tsx

```diff
@@ -0,0 +1,238 @@
+import React from 'react';
+import { TextField, Grid, Button, Switch, CircularProgress } from '@material-ui/core';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import { UserProfile } from '../../interface/Profile';
+import { Formik, FormikHelpers } from 'formik';
+import moment from 'moment';
+import useStyles from './useStyles';
+// const months = [
+//   'January',
+//   'February',
+//   'March',
+//   'April',
+//   'May',
+//   'June',
+//   'July',
+//   'August',
+//   'September',
+//   'October',
+//   'November',
+//   'December',
+// ];
+
+interface Props {
+  currentUserProfile: UserProfile;
+  handleProfileSubmit: (
+    { firstName, lastName, gender, birthDate, hourlyRate, address, email, description }: UserProfile,
+    { setSubmitting }: FormikHelpers<UserProfile>,
+  ) => void;
+}
+const cities = ['Toronto', 'Barrie', 'Markham', 'Richmood Hii'];
+const provinces = [
+  'Alberta',
+  'British Columbia',
+  'Manitoba',
+  'New Brunswick',
+  'Newfoundland and Labrador',
+  'Northwest Territories',
+  'Nova Scotia',
+  'Nunavut',
+  'Ontario',
+  'Prince Edward Island',
+  'Quebec',
+  'Saskatchewan',
+  'Yukon Territory',
+];
+const gender = ['male', 'female', 'unknow'];
+const EditProfile = ({ handleProfileSubmit, currentUserProfile }: Props): JSX.Element => {
+  const classes = useStyles();
+
+  return (
+    <div>
+      <h1 className={classes.profileTitle}>Edit Profile</h1>
+      <Formik
+        initialValues={{
+          firstName: currentUserProfile?.firstName || '',
+          lastName: currentUserProfile?.lastName || '',
+          gender: currentUserProfile?.gender || '',
+          email: currentUserProfile?.email || '',
+          birthDate: currentUserProfile?.birthDate || undefined,
+          hourlyRate: currentUserProfile?.hourlyRate || 14,
+          address: currentUserProfile?.address || { city: 'Toronto', province: 'Ontario' },
```

Comment:
Same here, assuming everyone is in Toronto by default is a bit odd, we should just keep it blank.